tests/unit/h/services/group_members_test.py

```diff
@@ -378,17 +378,35 @@ def test_it_proxies_to_member_join_and_leave(
             ]
         )
 
-    def test_it_does_not_add_duplicate_members(self, factories, group_members_service):
-        # test for idempotency
```

Comment:
This existing test confused me so I just fixed it up a bit with some clarifying comments.

Removed this existing comment because this is not really correct usage of "idempotent". What it's testing is a single call to `update_members(group, userids)` where the same user appears twice in `userid`'s.